modules/setting/setting.controller.js

```diff
@@ -2,4 +2,14 @@ const createError = require("http-errors");
 const settingModel = require("./setting.model");
 const logger = require("../../config/logLevels");
 
-module.exports = {};
+module.exports = {
+    getAllSettings: async (req, res, next) =>{
+        let result = await settingModel.getAllSettings(req.con);
+        if (result instanceof Error){
+            logger.error("Error en el modulo Setting (getAllSettings)");
```

Comment:
Hola, te sugiero que revises el módulo de user para que veas como se crea un error y se pasa por la función "next()". Debes agregar eso en esta parte para poder manejar el error correctamente!

```suggestion
            logger.error("Error en el modulo Setting (getAllSettings)");
            next(createError(status, "Indica el error más específicamente"))
```
Verifica el módulo que te mencioné anteriormente para que veas como crear "createError"